src/js/views/maps/viewfinder/ExpansionPanelView.js

```diff
@@ -0,0 +1,127 @@
+'use strict';
+
+define(
+  [
+    'underscore',
+    'backbone',
+    'text!templates/maps/viewfinder/expansion-panel.html',
+  ],
+  (_, Backbone, Template) => {
+    // The base classname to use for this View's template elements.
+    const BASE_CLASS = 'expansion-panel';
+
+    /**
+     * @class ExpansionPanelView
+     * @classdesc Allow expand and collapse content in a panel.
+     * @classcategory Views/Maps/Viewfinder
+     * @name ExpansionPanelView
+     * @extends Backbone.View
+     * @screenshot views/maps/viewfinder/ExpansionPanelView_closed.png
+     * @screenshot views/maps/viewfinder/ExpansionPanelView_open.png
```

Comment:
Only one of these screenshots will show up in the docs. Here's a single image you could use:

![ExpansionPanelView](https://github.com/NCEAS/metacatui/assets/26600641/af481610-f739-40d8-b652-f3a683998592)